pkg/esxi/esxi.go

```diff
@@ -90,6 +90,19 @@ func mountPartition(dev string) (options, error) {
 	return opts, nil
 }
 
+func loadBootImage(opts options, device string, partition int, ibft *ibft.IBFT) error {
```

Comment:
Does this work without an IBFT?

What if we don't ISCSI boot? I don't feel like the ibft belongs here in these functions, but I also don't see a better way to do it, so I'm ok as long as this works with a null ibft.

Reply:
Would you be able to test this without an ibft?

Reply:
Yeah, I did test it without an iBFT. And you can call this with nil. I kind of hate it, too, especially because the structure makes all of this kind of untestable. I need to figure out a better way.